app/views/dashboard/service_calendars/_view_full_calendar.html.haml

```diff
@@ -89,15 +89,15 @@
                 %h4.panel-title
                   = t(:dashboard)[:service_calendars][:consolidated_request][:services][:header]
               .panel-body
-                - if @protocol.has_per_patient_per_visit? @service_request, @portal
+                - if @protocol.has_line_items_of_type? @service_request, @portal
                   - @protocol.arms.each do |arm|
                     .arm_calendar_container.calendar-container{ data: { arm_id: arm.id } }
                       - next if arm.visit_groups.empty?
                       = render partial: '/dashboard/service_calendars/master_calendar/pppv/pppv_calendar', locals: { service_request: @service_request, arm: arm, page: @pages[arm.id], pages: @pages, review: false, merged: true, study_tracker: false }
-                - if @protocol.has_one_time_fees? @service_request, true
+                - if @protocol.has_line_items_of_type? @service_request, portal = true, one_time_fee = true
```

Comment:
AH! Not I see what you're saying. That was actually already like that, but you're right, I think. Looks like when this was originally written, they didn't realize that @portal was already a variable, and you could just pass that in, instead of "true."